caffeinated/modules/ticket_selector_caff/EED_Ticket_Selector_Caff.module.php

```diff
@@ -128,24 +124,17 @@ public static function _ticket_selector_settings_form()
      */
     public static function _ticket_selector_appearance_settings()
     {
-        if (! \EE_Registry::instance()->CFG->template_settings->EED_Ticket_Selector instanceof EE_Ticket_Selector_Config
+        if (! EE_Registry::instance()->CFG->template_settings->EED_Ticket_Selector instanceof EE_Ticket_Selector_Config
         ) {
-            \EED_Ticket_Selector::instance()->set_config();
-            \EE_Registry::instance()->CFG->template_settings->EED_Ticket_Selector = \EED_Ticket_Selector::instance(
+            EED_Ticket_Selector::instance()->set_config();
+            EE_Registry::instance()->CFG->template_settings->EED_Ticket_Selector = EED_Ticket_Selector::instance(
```

Comment:
I know this is just preserving the existing style aside from removing the root namespace but I think it'd be good to improve this by doing the linebreak on the `->` operator instead of the bracket.

Reply:
I didn't even notice that had happened... 
I think it may have been that way when I opened the file, but I agree... 
will change this